hr_employee_ssn/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     'category': 'Human Resources',
     'website': 'https://github.com/OCA/hr',
     'author':
-        'Braibean Apps, '
+        'Brainbean Apps, '
```

Comment:
could you also move the comma out of the quoted string?

Reply:
Sure, what's the guidelines for this field?

Reply:
Comma is correct as is. It's part of the string.

Reply:
my bad, the syntax was correct indeed sorry I forgot author was a single string.